testing/separated/places-bench/Cargo.toml

```diff
@@ -7,7 +7,7 @@ edition = "2021"
 publish = false
 
 [[bench]]
-name = "places"
+name = "places-bench"
```

Comment:
Had to rename the target to something else than `places`, because building the project sometimes caused an error when two crates have the same name (there is a `places` crate as well).